src/app/ApplicationTemplate.Shared/Presentation/Welcome/HomePageViewModel.cs

```diff
@@ -63,7 +61,7 @@ public partial class HomePageViewModel : ViewModel
 
 		public IDynamicCommand NavigateToChuckNorrisSearchPage => this.GetCommandFromTask(async ct =>
 		{
-			await this.GetService<IStackNavigator>().Navigate(ct, () => new ChuckNorrisSearchPageViewModel());
+			await this.GetService<IStackNavigator>().Navigate(ct, () => new ChuckNorrisSearchPageViewModel(() => this.GetService<IChuckNorrisService>()));
```

Comment:
Not sure we want to do this change, I think we explicitly decided each VM would resolve their own services.

@jeanplevesque any input?

Reply:
Yeah VMs have a ServiceProvider, so we shouldn't inject services.

Reply:
I understand but I have no choice to do this (so far) to inject my mocked service to my test.
I'll try to find a way.

Reply:
If not, you can also override the `ConfigureHost` method on `TestBase`.